src/View/SerializedView.php

```diff
@@ -127,7 +127,7 @@ function ($v) {
             } catch (Exception | TypeError $e) {
                 throw new SerializationFailureException(
                     'Serialization of View data failed.',
-                    null,
+                    0,
```

Comment:
This should stay `null` as the `Cake\Core\Exception\Exception` sets the error code to value of it's `$_defaultCode` property and if the argument is `null`.

Reply:
Ah, makes sense then.